pygmt/tests/test_datasets_earth_relief.py

```diff
@@ -30,6 +30,11 @@ def test_earth_relief_01d_igpp_synbath(data_source):
     data = load_earth_relief(
         resolution="01d", registration="gridline", data_source=data_source
     )
+    assert data.name == "elevation"
+    assert data.attrs["units"] == "meters"
+    assert data.attrs["long_name"] == "Earth elevation relative to the geoid"
+    assert data.attrs["vertical_datum"] == "EMG96"
```

Comment:
Typo, should be EGM96, see https://cddis.nasa.gov/926/egm96/egm96.html
```suggestion
    assert data.attrs["vertical_datum"] == "EGM96"
```